consensus/state.go

```diff
@@ -1681,13 +1681,18 @@ func (cs *State) recordMetrics(height int64, block *types.Block) {
 	cs.metrics.MissingValidators.Set(float64(missingValidators))
 	cs.metrics.MissingValidatorsPower.Set(float64(missingValidatorsPower))
 
-	cs.metrics.ByzantineValidators.Set(float64(len(block.Evidence.Evidence)))
+	// byzantine validators power and count is only for consensus evidence i.e. duplicate vote
```

Comment:
can we include light client evidence here?

Reply:
We don't know from the light client evidence alone how many validators were malicious (we would need the evidence pool to work it out - which could be possible as we have access to it but we would need to shuffle the code around a bit)